Tests/UI/CvvInputViewTests.swift

```diff
@@ -4,7 +4,7 @@ import XCTest
 class CvvInputViewTests: XCTestCase {
 
     var cvvInputView: CvvInputView!
-    var stubCardValidator: StubCardValidator! = StubCardValidator()
+    var stubCardValidator: MockCardValidator! = MockCardValidator()
```

Comment:
```suggestion
    var stubCardValidator = MockCardValidator()
```

Reply:
This logic and that in `tearDown()` are used to automatically reinit between tests. This is a pattern across Frames and 3DS